scripts/ci/ci_before_install.sh

```diff
@@ -33,6 +33,7 @@ KUBERNETES_MODE=${KUBERNETES_MODE:=""}
 
 mkdir -p "${AIRFLOW_SOURCES}/files"
 
-sudo pip install pre-commit
+# We need to install six apparently
```

Comment:
Technically this isn't correct since six is already installed, but an older (and incompatible) version:
```
ERROR: virtualenv 20.0.0 has requirement six<2,>=1.12.0, but you'll have six 1.11.0 which is incompatible.
```